packages/cli/src/commands/worker.ts

```diff
@@ -267,9 +267,9 @@ export class Worker extends BaseCommand {
 	}
 
 	async init() {
-		const configuredShutdownTimeout = config.getEnv('queue.bull.gracefulShutdownTimeout');
-		if (configuredShutdownTimeout) {
-			this.gracefulShutdownTimeoutInS = configuredShutdownTimeout;
+		const shutdownTimeoutFromDeprecatedEnv = config.getEnv('queue.bull.gracefulShutdownTimeout');
+		if (shutdownTimeoutFromDeprecatedEnv !== config.default('queue.bull.gracefulShutdownTimeout')) {
```

Comment:
Maybe I'm confused but if `shutdownTimeoutFromDeprecatedEnv` is `config.getEnv('queue.bull.gracefulShutdownTimeout')` wouldn't that mean that `shutdownTimeoutFromDeprecatedEnv !== config.default('queue.bull.gracefulShutdownTimeout')` will always be false?

Reply:
`config.getEnv('queue.bull.gracefulShutdownTimeout')` resolves to its default value `30` when unspecified. We only want to override and warn if the user is using the deprecated env.

Reply:
My bad, I was confused and missed that `getEnv` is not `default`. 😄